uint8 pause
---
bool result
